wunderio/drupal-project
# #318 Add ESLint support and configure it to use Drupal standards.

*How to test:*

1. Get this branch
    `git clone git@github.com:wunderio/drupal-project.git`
    `cd drupal-project`
    `git checkout feature/#318-Add-ESLint-support-and-configure-it-to-use-Drupal-standards`
2. Start Lando
   ` lando start` (or `lando rebuild` if you've working on existing Lando project)
3. Create example js file
    `echo 'var foo="bar"' > web/modules/custom/phpunit_example/example.js`
4. Add the file to GIT
    `git add web/modules/custom/phpunit_example/example.js`
5. Try to commit
    `git commit`
6. Allow it to fix the script automatically and check the results


There's also an alternative PR here https://github.com/wunderio/drupal-project/pull/320 that uses node service but it's currently proof of work. It breaks Code Quality scans, you need to configure name and e-mail in the node service.
